packages/web/src/scss/helpers/_scrollable-behavior.scss

```diff
@@ -0,0 +1,5 @@
+.is-scrolling-disabled:is(body) {
```

Comment:
I think the `body` here is not necessary, is it? In the future, we may extend the scroll blocking for elements other than `body` anyway.

Reply:
done